src/tribler/core/utilities/tracker_utils.py

```diff
@@ -62,7 +62,7 @@ def get_uniformed_tracker_url(tracker_url):
         # accessing urlparse attributes may throw UnicodeError's or ValueError's
         try:
             # scheme must be either UDP or HTTP
-            if url.scheme == 'udp' or url.scheme == 'http':
+            if url.scheme == 'udp' or url.scheme == 'http' or url.scheme == 'https':
                 uniformed_scheme = url.scheme
```

Comment:
**nitpick:** `uniformed_scheme` variable looks unnecessary; we can just use `url.scheme` everywhere

Reply:
Refactored it differently now.